### PR TITLE
readme: update maintained versions table

### PR DESCRIPTION
**Pull request by muvaf**
Deprecated v1.1 since it's EOL is reached.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,14 +51,14 @@ documentation].
 
 | Release |  Current Patch  | Release Date |      EOL      |
 |:-------:|:---------------:|:------------:|:-------------:|
-|   v1.1  |     [v1.1.4]    |  Mar 3, 2021 |  August 2021  |
 |   v1.2  |     [v1.2.4]    | Apr 27, 2021 | October 2021  |
 |   v1.3  |     [v1.3.1]    | Jun 29, 2021 | December 2021 |
-|   v1.4  |     Upcoming    | Aug 31, 2021 | February 2022 |
+|   v1.4  |     [v1.4.0]    | Aug 31, 2021 | February 2022 |
+|   v1.5  |     Upcoming    | Oct 26, 2021 | April 2022 |
```
Comment (hasheddan) [inline README.md:57]: nitpick:
```suggestion
|   v1.4  |     [v1.4.0]    | Aug 31, 2021 | February 2022 |
|   v1.5  |     Upcoming    | Oct 26, 2021 |   April 2022  |
```

**Review by hasheddan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (muvaf) [reply README.md:57]: It shows up as centered in preview actually and I see others lean to left.

Comment (hasheddan) [reply README.md:57]: Hmm, I think that must have happened in a recent update -- can we keep column width consistent?

Comment (muvaf) [reply README.md]: Done!

**Pull request merged**